.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "stable"
+install:
+  - npm install
```

Comment:
Can you try with `npm run build` to see if the build pass?

Reply:
done